### ISSUE TITLE
All dice prefix commands not working

### ISSUE THREAD

**Issue by otakuo402** [Rolisteam/DiceParser#148]
I have a server that uses diceparcer for DnD and none of my players nor me an admin can get the prefix commands to work. The /roll command works but it would require everyone to be an admin for them to use the command which just isn't possible at this moment. I checked with another DM and he said that he was seeing the same issue in his server.

**Comment by rwestbrock** [comment]
I saw in another thread that you may have to kick the bot and re-invite it which means you would need to re-grant permissions to it. I thin you have to re-create aliases as well.

If by prefix you mean the old ! then that won't work any longer from what I read.

**Comment by rwestbrock** [comment]
This is the issue I was referring to in my previous comment: https://github.com/Rolisteam/DiceParser/issues/147

**Comment by obiwankennedy** [comment]
yes, kick out the bot and re-invite it from this link: https://discord.com/api/oauth2/authorize?client_id=279722369260453888&permissions=277025429504&scope=applications.commands%20bot 

At least, after that, you should see diceparser and its commands in the menu appearing when you type '/'.
If some user can see them, check the permission of evething.
